core/src/main/java/de/eldoria/bloodnight/mob/meta/ExtensionType.java

```diff
@@ -1,5 +1,5 @@
 package de.eldoria.bloodnight.mob.meta;
 
 public enum ExtensionType {
-    NONE, RIDER, CARRIER
+    RIDER, CARRIER
```

Comment:
We need none. Its much cleaner.
```suggestion
    NONE, RIDER, CARRIER
```